src/query-helpers.js

```diff
@@ -2,33 +2,8 @@ import {prettyDOM} from './pretty-dom'
 import {fuzzyMatches, matches, makeNormalizer} from './matches'
 import {waitForElement} from './wait-for-element'
 
-/* eslint-disable complexity */
-function debugDOM(htmlElement) {
-  const limit = process.env.DEBUG_PRINT_LIMIT || 7000
```

Comment:
do we want to support this env var still?

Reply:
My changes keep it. Are you suggesting we should remove it?

Reply:
nope, I see it's moved into prettyDOM itself now 👍